src/Modules/Isrs.asm

```diff
@@ -392,7 +405,61 @@ ELSEIF PWM_BITS_H == PWM_10_BIT
     rrc  A
     cpl  A
     mov  Temp2, A
-ELSEIF PWM_BITS_H == PWM_9_BIT
+
+    ; Set PCA to work at 48khz
+    mov  PCA0PWM, #82h
+ELSE
+t1_int_run_24khz:
+    ; Scale pwm resolution and invert (duty cycle is defined inversely)
+    ; Deadtime and 24khz
+    clr  C
+    mov  A, Temp5
+    rrc  A
+    cpl  A
+    anl  A, #3
+    mov  Temp3, A
+    mov  A, Temp4
+    rrc  A
+    cpl  A
+    mov  Temp2, A
+
+    ; Set PCA to work at 24khz
+    mov  PCA0PWM, #82h
+
+    ; Subtract dead time from normal pwm and store as damping PWM
+    ; Damping PWM duty cycle will be higher because numbers are inverted
+    clr  C
+    mov  A, Temp2                       ; Skew damping FET timing
+IF MCU_TYPE == MCU_BB1
```

Comment:
Should we still account for BB1 here? We have dropped it officially - so I think we'd be fine to no longer consider it in newly implemented features.

Reply:
I would like to remove BB1 on a separate commit, if possible on the next release

Reply:
OK, makes sense, moved to #150